core/src/main/java/org/opensearch/sql/utils/DateTimeUtils.java

```diff
@@ -151,4 +151,6 @@ public static LocalDate extractDate(ExprValue value,
         ? ((ExprTimeValue) value).dateValue(functionProperties)
         : value.dateValue();
   }
+
+  public static final ZoneId UTC_ZONE_ID = ZoneId.of("UTC");
```

Comment:
Should constants be at the beginning or at the end?

Reply:
Do we also want to "encourage" private functions to be defined after public functions?